scripts/ci/publishing-needed.js

```diff
@@ -0,0 +1,49 @@
+import path, { dirname } from "path";
+import fs from "fs/promises";
+import { spawnSync } from "child_process";
+import { fileURLToPath } from "url";
+
+if (process.argv.length < 3) {
+  throw new Error(`Usage: node publishing-needed.js <name-of-the-workspace>`);
+}
+const workspace = process.argv[2];
+
+(async function main() {
+  const pluginsFolderPath = path.join(
+    dirname(fileURLToPath(import.meta.url)),
```

Comment:
is this preferred over __dirname etc?

Reply:
we are in ESM land so `__dirname` won't be available